tutorials/shading/migrating_to_godot_shader_language.rst

```diff
@@ -0,0 +1,211 @@
+.. _doc_migrating_to_godot_shader_language:
+
+Migrating to Godot shading language
+===================================
+
+Introduction
+------------
+
+This document will explain the differences between Godot's shading language
+and glsl as well as give practical advice on how to migrate shaders from other
+sources, such as Shadertoy and The Book of Shaders, into Godot shaders.
+
+For detailed information on Godot's shading language please refer to the :ref:`Shading Language <doc_shading_language>`
+reference.
+
+Glsl
+----
+
+Godot uses a shading language based on glsl, with the addition of a few quality of life features.
```

Comment:
Remove comma

When used as an adjective, QOL is usually hyphenated: "quality-of-life".